spec/serializers/snippet_version_serializer_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe SnippetVersionSerializer do
+  let(:snippet) { create(:snippet) }
+  let(:version) { create(:version, versionable: snippet) }
+  let(:serializer) { described_class.new(version).as_json }
+
+  before(:each) do
+    allow(Snippet).to receive(:find).with(anything) { snippet }
```

Comment:
what is this `anything` argument you are passing here?

Reply:
http://rspec.info/documentation/3.4/rspec-mocks/RSpec/Mocks/ArgumentMatchers.html#anything-instance_method :)